packages/lesswrong/components/vulcan-forms/Form.tsx

```diff
@@ -266,7 +266,7 @@ class Form extends Component<any,any> {
   */
   getFieldGroups = () => {
     // build fields array by iterating over the list of field names
-    let fields: Array<any> = this.getFieldNames().map((fieldName: string) => {
+    let fields: Array<any> = this.getFieldNames(this.props).map((fieldName: string) => {
```

Comment:
I'm a bit nervous to be making changes to `Form.tsx`, will prob do some more testing. Here I needed to pass in the `props` so that I could use `excludeHiddenFields` in my new form.

Reply:
~You could make this change safer by purposefully extracting out `excludeHiddenFields` from props and passing it in.~

I do like the generality of this. You could look at all the places we call WrappedSmartForm and see whether any of them pass in anything that could have been plucked out on line 315 from args. Actually, I just did this, and am confident it's fine. Manual QA is also good, but I think this is a pretty safe change